velox/exec/Task.h

```diff
@@ -69,14 +69,17 @@ class Task : public std::enable_shared_from_this<Task> {
   /// thread are passed on to a separate consumer.
   /// @param onError Optional callback to receive an exception if task
   /// execution fails.
+  /// @param memoryArbitrationPriority Optional priority on task that, in a
```

Comment:
Let's be specific for memory arbitration for now?

Reply:
yeah